prisma/migrations/20240117191553_categories_for_teller/migration.sql

```diff
@@ -0,0 +1,223 @@
+-- AlterTable
+BEGIN;
+
+ALTER TABLE
+  "transaction" RENAME COLUMN "category" TO "category_old";
+
+ALTER TABLE
+  "transaction" RENAME COLUMN "category_user" TO "category_user_old";
+
+DROP VIEW IF EXISTS transactions_enriched;
+
+ALTER TABLE
+  "transaction"
+ADD
+  COLUMN "category_user" TEXT;
+
+ALTER TABLE
+  "transaction"
+ADD
+  COLUMN "category" TEXT NOT NULL GENERATED ALWAYS AS(
```

Comment:
I am not a fan of these generated expressions living in the database, I would prefer to move all of this logic into the ETL path in the future so that changes don't end up causing all sorts of migration issues. A future task and QOL improvement would be to rip out all these dbgenerated expressions from the database and schema and port them to server code.